commons/src/main/scala/com/lynbrookrobotics/potassium/commons/Value3D.scala

```diff
@@ -0,0 +1,49 @@
+package com.lynbrookrobotics.potassium.commons
+
+import squants.Quantity
+
+/**
+  * Constructs a new 3D value given X, Y, and Z axes.
+  */
+case class Value3D[Q <: Quantity[Q]](x: Q, y: Q, z: Q) {
```

Comment:
Can you move this to core, since we seem to be using it in a lot of different places?